template/.electron-vue/build.js

```diff
@@ -46,32 +46,49 @@ function build () {
 
   let results = ''
 
-  m.on('success', () => {
-    process.stdout.write('\x1B[2J\x1B[0f')
-    console.log(`\n\n${results}`)
-    console.log(`${okayLog}take it away ${chalk.yellow('`electron-{{builder}}`')}\n`)
-    {{#if_eq builder 'packager'}}bundleApp(){{else}}process.exit(){{/if_eq}}
-  })
-
-  pack(mainConfig).then(result => {
-    results += result + '\n\n'
-    m.success('main')
-  }).catch(err => {
-    m.error('main')
-    console.log(`\n  ${errorLog}failed to build main process`)
-    console.error(`\n${err}\n`)
-    process.exit(1)
-  })
-
-  pack(rendererConfig).then(result => {
-    results += result + '\n\n'
-    m.success('renderer')
-  }).catch(err => {
-    m.error('renderer')
-    console.log(`\n  ${errorLog}failed to build renderer process`)
-    console.error(`\n${err}\n`)
-    process.exit(1)
-  })
+  const tasks = new Listr(
```

Comment:
already has var named `tasks`
const tasks = ['main', 'renderer']

Reply:
Yes why a such mistake, alll the build pipeline is broken now.
How could this have been merged into master?